### PR TITLE
Implement Class Mastery page

### PR DESCRIPTION
**Pull request by nguyen-darianhuy**
Summary
---
- [x] Implemented Class Mastery design and filled it with mock data
- [ ] Connect it to backend
- [ ] Provide a clickable link to it on another page

Screen Shot 2021-05-12 at 3 29 51 PM

### CODE REVIEW

File: src/Components/Content/Content.tsx [hunk 1]
```diff
@@ -7,9 +7,9 @@ import ViewTaskSubmission from '../ViewTaskSubmission';
 import SingleStudentOverview from '../SingleStudentOverview/SingleStudentOverview';
```
Comment (mattcuento) [inline src/Components/Content/Content.tsx:7]: Create index files within each Component folder to avoid repetition of path names

Comment (mattcuento) [reply src/Components/Content/Content.tsx:7]: Ditto for the components below

File: src/Screens/ClassMastery/ClassMastery.tsx [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+   font-size: 36px;
+   font-weight: 550;
+   margin-bottom: 32px;
+`;
+
+const Content = styled.div``;
+
+const MOCK_LOBJ_NAMES = ['Objective 1', 'Objective 2'];
+
+const MOCK_DATA = [
```
Comment (mattcuento) [inline src/Screens/ClassMastery/ClassMastery.tsx:21]: Abstract mock data to the mock folder

**Review by mattcuento — CHANGES_REQUESTED** [review]
Create index files to avoid repitition of path names in import statements, good other than that

**Review by rmfay1 — COMMENTED** [review]
complete matt's requested changes

**Comment by nguyen-darianhuy** [comment]
Addressed changes

**Pull request merged**